lib/krikri/harvester.rb

```diff
@@ -77,5 +84,40 @@ def run
       true
     end
 
+    ##
+    # An application-wide registry of defined Harvesters.
+    class Registry
```

Comment:
What are the differences between this and the mapping registry? How much of this, and the related tests, is duplicated from there?